src/send/test.rs

```diff
@@ -8,9 +8,19 @@ use rocket::{
 use serde_json;
 
 use app_errors::{self, ApplicationError};
+use auth_db::DbClient;
+use bounces::Bounces;
+use providers::Providers;
+use settings::Settings;
 
 fn setup() -> Client {
+    let settings = Settings::new().unwrap();
+    let db = DbClient::new(&settings);
+    let bounces = Bounces::new(&settings, db);
+    let providers = Providers::new(&settings);
```

Comment:
In wonder if `providers` or some othe struct need `db` in the future, would this work or not:

```Rust
let bounces = Bounces::new(&settings, db);
let providers = Providers::new(&settings, db);
let thins = Things::new(&settings, db);
```

**shrugs**

Reply:
Fwiw, if `providers` needs access to `auth_db` it means something has gone seriously wrong in our separation of concerns. :smile:

The only thing this repo should have to do with the auth db is reading bounce records and writing bounce records, and even those are temporary until we migrate to a separate data store. Restricting access to where it's needed is a feature, imho.

Reply:
(in that sense, the `bounces` module represents our bespoke interface for the db, we want to hide everything else behind it)

Reply:
(and writing this makes me realise the queues process should also be accessing it via bounces too now) 

😄